internal/app/machined/pkg/system/services/kubelet.go

```diff
@@ -105,6 +105,10 @@ func (k *Kubelet) PreFunc(ctx context.Context, r runtime.Runtime) error {
 		return err
 	}
 
+	if err := os.MkdirAll("/run/kubelet", 0700); err != nil {
```

Comment:
nit: Can we do `/run/system/kubelet`? That is where we have other service directories.

Reply:
copying from Slack for history:

> I thought /run/system is for system services vs. k8s stuff which lives in /run ?
> e.g. CRI containerd is /run/containerd/, but system containerd is /run/system/containerd

Reply:
:+1: